x/staking/genesis.go

```diff
@@ -237,17 +237,22 @@ func validateGenesisStateValidators(validators []types.Validator) error {
 		if err != nil {
 			return err
 		}
-		consAddr, err := val.GetConsAddr()
-		if err != nil {
-			return err
-		}
+
 		strKey := string(consPk.Bytes())
 
 		if _, ok := addrMap[strKey]; ok {
+			consAddr, err := val.GetConsAddr()
```

Comment:
I wonder can we just skip the err here and just do `consAddr, _ := val.GetConsAddr()`?

Reply:
Nah, let's keep it as is.